src/main/java/com/RegisterQuizApplication/Registration/ResultRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.RegisterQuizApplication.Registration;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface ResultRepository extends JpaRepository<Result,Integer> {
+    @Query(
```

Comment:
similar comment. Avoid @Query and try to use the builtIn method naming convention

https://docs.spring.io/spring-data/jpa/reference/jpa/query-methods.html